src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int maxNumber = 100;

```

Comment:
```suggestion
    private static final int maxNumber = 100;
```

the same, upper-case letter